src/features/bookings/pages/BookingDetails/BookingDetails.tsx

```diff
@@ -76,9 +77,10 @@ export function BookingDetails() {
 
   const mapping = useSubcategoriesMapping()
 
-  const { venue, id: offerId } = booking?.stock.offer ?? {}
-  const { address, postalCode, city } = venue ?? {}
-  const venueFullAddress = address ? formatFullAddress(address, postalCode, city) : undefined
+  const { id: offerId, address } = booking?.stock.offer ?? {}
+  const offerFullAddress = address
```

Comment:
Dans le cas où on n'a pas `address` dans l'offer, ne voudrait-on pas fallback sur `venue` (pour le moment) ? Ou est-ce qu'on est ok côté métier pour ne rien afficher si c'est le cas ?

Reply:
J'ai vu avec Henri on est ok pour ne rien afficher si on n'a pas l'info (mais elle semble bien renseigner sur staging déjà)

Reply:
J'en ai profité pour ajouter un commit en éliminant quelques @ts-expect-error: because of noUncheckedIndexedAccess ^^